plugins/airbrake-backend/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@backstage/plugin-airbrake-backend",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "start": "backstage-cli backend:dev",
+    "build": "backstage-cli backend:build",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "^0.10.5",
+    "@backstage/config": "^0.1.13",
+    "@types/express": "*",
+    "express": "^4.17.1",
+    "express-promise-router": "^4.1.0",
+    "http-proxy-middleware": "^2.0.0",
+    "winston": "^3.2.1",
+    "cross-fetch": "^3.0.6",
+    "yn": "^4.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.13.1-next.0",
+    "@types/http-proxy-middleware": "^0.19.3",
+    "@types/supertest": "^2.0.8",
+    "supertest": "^4.0.2",
```

Comment:
Can you bump this to `supertest@^6.1.6` as it's what we've got in the project already. saves downloading two different versions.